tests/testthat/test_SpatialExperiment-spatialCoords.R

```diff
@@ -0,0 +1,36 @@
+example(read10xVisium, echo = FALSE)
+
+test_that("spatialCoordsNames()", {
+    expect_identical(
+        spatialCoordsNames(spe), 
+        colnames(int_colData(spe)$spatialCoords))
+})
+
+test_that("spatialCoordsNames<-,character", {
+    old <- spatialCoordsNames(spe)
+    new <- sample(letters, length(old))
+    spatialCoordsNames(spe) <- new
+    expect_identical(spatialCoordsNames(spe), new)
+    expect_identical(spatialCoordsNames(spe), 
+        colnames(int_colData(spe)$spatialCoords))
+})
+
+test_that("spatialCoordsNames<-,NULL", {
+    old <- spatialCoords(spe)
+    spatialCoordsNames(spe) <- NULL
+    expect_null(spatialCoordsNames(spe))
+    expect_equivalent(spatialCoords(spe), old)
+})
+
+test_that("spatialCoords<-,matrix", {
+    # no 'rownames(value)' passes
+    rownames(xyz) <- NULL
```

Comment:
there are no xyz in the read10xVisium example, I suppose the GHA error comes from here...

Reply:
Damn it, I had reversed the order of some lines... should be fixed now.